corehq/apps/ota/tests/test_app_aware_restore.py

```diff
@@ -68,6 +71,7 @@ def setUpClass(cls):
         cls.app2.save()
 
         cls.app3 = Application.new_app(cls.domain, 'Test App 3')
+        cls.app3.mobile_ucr_restore_version = MOBILE_UCR_VERSION_1
```

Comment:
do these these tests fail if its V2? Since we are defaulting to V2, I believe tests should use V2 unless V1 is specifically mentioned, no?

Reply:
> do these these tests fail if its V2? 

That is correct as some of the tests explicitly checks for the expected output based on v1.

> Since we are defaulting to V2, I believe tests should use V2 unless V1 is specifically mentioned, no?

Good question. Since we are still keeping the v1 and v1.5 code references for now (planned to clean up at a later stage), I figured this PR might not be the best place to address this. However I don't have a strong preference and also good with updating these now considering the changes are not that big.

Reply:
For tests, I believe, if anything is to break by being on V2, we should know that now. So, updating tests to use V2 everywhere, sounds like one good way to know that.

Reply:
Beautiful, thank you @ajeety4. That was quite some effort but it was worth it :1st_place_medal:

Reply:
Thanks for catching. Updated :+1: